components/todo-item.js

```diff
@@ -1,11 +1,50 @@
-import React from "react";
-import styled from "styled-components";
+import React from 'react';
+import styled from 'styled-components';
 
-export default function TodoItem() {
-  return <Wrapper>Todo</Wrapper>;
+export default function TodoItem(props) {
+	const { id, date, todo, onRemove } = props;
+
+	return (
+		<TodoItemWrapper>
+			<Todo>{todo}</Todo>
+			<Date>
+				{date}
+				<Button onClick={() => onRemove(id)}>완료</Button>
+			</Date>
+		</TodoItemWrapper>
+	);
 }
+const Date = styled.div`
+	font-size: 1.5rem;
+	display: flex;
+	color: white;
+	flex-direction: column;
+`;
+const Todo = styled.div`
+	font-size: 1.5rem;
+	color: white;
+	display: flex;
+	flex-direction: column;
+`;
+const Button = styled.button`
+	color: white;
+	background-color: rgb(97, 97, 97);
+	font-size: 1.5rem;
+	outline: none;
+	border-width: initial;
+	border-style: none;
+	padding: 0.5rem 1rem;
+	border-radius: 0.3rem;
+`;
```

Comment:
```suggestion
const Row = styled.div`
      display: flex;
      flex-direction: row;
      justify-content: space-between
`;
const P = styled.p`
	font-size: 1.5rem;
	color: white;
`;
const DeleteButton = styled.button`
	color: white;
	background-color: rgb(97, 97, 97);
	font-size: 1.5rem;
	outline: none;
	border: none;
	border-radius: 0.3rem;
	padding: 0.5rem 1rem;
	width: fit-content;
	margin-left: auto;
`;
```
1. ```Date, Todo```의 style이 겹치네요! ```p``` 태그로 P 선언해서 써주시면 될 것 같아요 :)
2. 버튼의 역할을 정확하게 나타내기 위해 ```DeleteButton```으로 구체화하면 좋을 것 같아요~
3. 결과 화면에서 ```TodoItem```의 구조를 살펴보면,
첫 번째 행 양 끝에 내용과 날짜가 있고, 그 다음 행 오른쪽 끝에 완료 버튼이 있죠!
그래서 ```Date``` 태그 안에 완료 버튼을 넣는 게 어색하게 느껴져요. 
내용, 날짜를 나타내는 태그를 묶으면서 한 행에 정렬하는 ```Row```태그 선언한 후 그 안에 ```P```태그들 넣어주시고,
DeleteButton에 ```margin-left: auto```를 주어 화면과 같은 스타일을 구현할 수 있어요. [참고](https://developer.mozilla.org/ko/docs/Web/CSS/margin-left#Values)